docker-compose.yml

```diff
@@ -13,7 +14,7 @@ services:
     
   central-ledger:
     image: mojaloop/central-ledger:latest
-    container_name: central-ledger
+    container_name: ml_central-ledger
```

Comment:
Why did you change the container name?

Reply:
Because it clashes with the container names for other environments running locally on docker-compose. 

The links still work just the same (you can refer to `central-ledger` for example when linking conatiners together), it's just when you want to look at the logs or run `docker exec` that this is a problem.